Source/SwiftLintFramework/Rules/MissingDocsRule.swift

```diff
@@ -54,17 +54,9 @@ extension File {
 }
 
 public enum AccessControlLevel: String {
```

Comment:
> Does this need to be public?

I don't know, it was already public, I didn't change this line, just changed the raw values.